JS/Lesson15/gulcin/script.js

```diff
@@ -113,52 +117,63 @@ console.log('------------------------');
 // 16. Extract the price and currency from a string below using "slice" method and print it to console.
 const planPriceString = 'Premium plan - 9.99 USD/month.'; // Should return "9.99 USD"
 
-console.log('Exercise16: ' /* Result here */);
+console.log('Exercise16: ' , planPriceString.slice(planPriceString.indexOf('9'), -7));
```

Comment:
Replace the "-7" with a more reliable solution that will look for the index of "/", for example